cfme/tests/services/test_provision_stack.py

```diff
@@ -197,21 +36,18 @@
 @pytest.yield_fixture(scope="function")
 def template(provider, provisioning, dialog_name):
     template_type = provisioning['stack_provisioning']['template_type']
-    if provider.type == 'azure':
-        template_name = 'azure-single-vm-from-user-image'
-    else:
-        template_name = fauxfactory.gen_alphanumeric()
-
+    template_name = fauxfactory.gen_alphanumeric()
     template = OrchestrationTemplate(template_type=template_type,
                                      template_name=template_name)
 
     if provider.type == "ec2":
```

Comment:
Azure is a special template which really can't be created on the fly.  Plus, in the future we're going to want more templates for error checking scenarios as well.  And that's just for VMs.  There are all kinds of orchestration templates that aren't being created.  Networks, storage, security groups.  I see what you're saying, but that's why I added _vm_ to the template name in anticipation of future scaling.